packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -404,7 +412,7 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		// The 'IDocumentStorageService.createBlob()' call below will respond with a localId.
 		const response = await this.getStorage().createBlob(blob);
 		this.setRedirection(response.id, undefined);
-		return this.getBlobHandle(response.id);
+		return this.getBlobHandle(response.id, undefined);
```

Comment:
why would be pass the remote id as the local id?

Reply:
this looks like a bug to me

Reply:
This is for detached scenario. Once attached we will have a "real" storageId. Basically creating a blob while detached creates a local id and behaves the same way as when stashing. You have a local only id which will later have a storage id

Reply:
And at the end it really won't have any difference in behaviour inside getBlobHandle whether we send it as the remote one. The change in getBlobHandle is basically just prioritize remote id over local id when we have both